src/lib/meta/meta_blk_service.cpp

```diff
@@ -196,9 +196,14 @@ void MetaBlkService::format_ssb() {
 // m_meta_lock should be while calling this function;
 void MetaBlkService::write_ssb() {
     // write current ovf blk to disk;
-    try {
-        m_sb_vdev->sync_write((const char*)m_ssb, block_size(), m_ssb->bid);
-    } catch (std::exception& e) { HS_REL_ASSERT(false, "exception happen during write {}", e.what()); }
+    auto error = m_sb_vdev->sync_write((const char*)m_ssb, block_size(), m_ssb->bid);
```

Comment:
How about create a new function `MetaBlkService::sync_write` and move all error handling to that function, to avoid duplicated error handling?
```
void MetaBlkService::sync_write(const char* data, uint32_t size, BlkId const& bid) { 
        // copy line: 199~206 here 
}
```

Reply:
if we do that , we also have to handle the the error returned by `MetaBlkService::sync_write` whereever it is called. that is almost the same as we use `m_sb_vdev->sync_write` directly

Reply:
MetaBlkService::sync_write::sync_write will return void and only error handling happens in MetaBlkService::sync_write::sync_write, not in the caller.

Reply:
ok , will do this in another PR